src/components/TransactionHistoryRows/TransactionGenericRow.tsx

```diff
@@ -111,222 +36,170 @@ export const TransactionGenericRow = ({
-              flex={1}
-              spacing={0}
-              fontSize={stackFontSize}
-              alignItems={stackAlignItems}
-            >
-              <Flex alignItems='center' gap={2} flex={1}>
-                <Text
-                  fontWeight='bold'
-                  translation={title ? title : `transactionRow.${type.toLowerCase()}`}
-                />
-                {txData && txData.parser === 'ibc' && (
-                  <Tag
-                    size='sm'
-                    colorScheme='blue'
-                    variant='subtle'
-                    minHeight={tagMinHeight}
-                    px={tagPx}
-                  >
-                    IBC
-                  </Tag>
-                )}
-                {isNft && (
-                  <Tag
-                    size='sm'
-                    colorScheme='blue'
-                    variant='subtle'
-                    minHeight={tagMinHeight}
-                    px={tagPx}
-                  >
-                    NFT
-                  </Tag>
-                )}
-                {txData && txData.parser === 'thorchain' && txData.liquidity && (
-                  <Tag
-                    size='sm'
-                    colorScheme='green'
-                    variant='subtle'
-                    minHeight={tagMinHeight}
-                    px={tagPx}
-                  >
-                    {txData.liquidity.type}
-                  </Tag>
-                )}
-                {txData && txData.parser === 'thorchain' && txData.swap?.type === 'Streaming' && (
-                  <Tag
-                    size='sm'
-                    colorScheme='green'
-                    variant='subtle'
-                    minHeight={tagMinHeight}
-                    px={tagPx}
-                  >
-                    {txData.swap.type}
-                  </Tag>
-                )}
-              </Flex>
-              <TransactionTime blockTime={blockTime} format={dateFormat} />
-            </Stack>
-          </Flex>
-        </Flex>
-        <Flex flex={2} flexDir='column' width='full'>
-          <Stack
-            direction='row'
-            width='full'
-            alignItems='center'
-            spacing={stackSpacing}
-            justifyContent={stackJustifyContent}
-            fontSize={stackFontSize}
-            divider={stackDivider}
-          >
-            {transfers}
-          </Stack>
+  const topLeft = useMemo(() => {
+    return (
+      <Text
+        color='text.subtle'
+        translation={title ? title : `transactionRow.${type.toLowerCase()}`}
+      />
+    )
+  }, [title, type])
+
+  const topRight = useMemo(() => {
+    if (hasManySends) {
+      const assets = transfersByType.Send.map(transfer => transfer.asset.symbol)
+      return <RawText color='text.subtle'>{assets.join('/')}</RawText>
+    }
+    if (hasSendAndRecieve) {
+      const precision = transfersByType.Send[0].asset.precision ?? FALLBACK_PRECISION
```

Comment:
We don't want to fallback to an arbitrary precision, and return potentially wrong amounts, just fallback to 0